lntest/itest/lnd_channel_backup_test.go

```diff
@@ -120,10 +120,17 @@ func testChannelBackupRestore(net *lntest.NetworkHarness, t *harnessTest) {
 				// Create a new nodeRestorer that will restore
 				// the node using the Multi backup we just
 				// obtained above.
+				oldPortOpts := func(cfg *lntest.NodeConfig) {
```

Comment:
nit: to avoid copy-paste you could also create a function returning a `NodeOption`, something like:
```
func copyNodePorts(oldNode *HarnessNode) NodeOption {
    return func(cfg *lntest.NodeConfig) {
		cfg.P2PPort = oldNode.Cfg.P2PPort
		cfg.RPCPort = oldNode.Cfg.RPCPort
		cfg.RESTPort = oldNode.Cfg.RESTPort
		cfg.ProfilePort = oldNode.Cfg.ProfilePort
	}
}
```

Reply:
Great idea, done.